src/core_seaice/Registry.xml

```diff
@@ -3490,6 +3503,17 @@
 		<var name="oceanHeatFluxIceBottom"		type="real"	dimensions="nCells Time"		name_in_code="oceanHeatFluxIceBottom"/>
 	</var_struct>
 
+	<!-- fluxes from data icebergs -->
+	<var_struct name="iceberg_forcing" time_levs="1">
+		<var name="bergFreshWaterFluxData"		type="real"	dimensions="nCells Time"		name_in_code="bergFreshWaterFluxData"/>
+	</var_struct>
+
+	<!-- fluxes from icebergs to ocean -->
+	<var_struct name="iceberg_fluxes" time_levs="1" packages="pkgColumnPackage">
+		<var name="bergFreshWaterFlux"			type="real"	dimensions="nCells Time"		name_in_code="bergFreshWaterFlux"/>
```

Comment:
@akturner - do you think this should be part of the pkgColumnPackage package (or a new package pkgBergs?)